tests/dualtor/test_tunnel_memory_leak.py

```diff
@@ -176,7 +182,7 @@ def remove_neighbor(duthost, server_ip):
                 pytest_assert(validate_neighbor_entry_exist(upper_tor_host, server_ipv4),
                             "The server ip {} doesn't exist in neighbor table on dut {}. \
                             tunnel_packet_handler isn't triggered.".format(server_ipv4, upper_tor_host.hostname))
-
+                time.sleep(3)
```

Comment:
Why add this delay?

Reply:
I added it for 8102 platform, but even sleep 3, it still always fails. Doesn't have to add it, remove it in last commit.